doc/api/Host.md

```diff
@@ -1,452 +1,536 @@
+    "maxrevisebatchsize": 17825792, // bytes
+
+    // The IP address or hostname (including port) that the host should be
+    // contacted at.
+    "netaddress": "123.456.789.0:9982",
+
+    // The amount of unused storage capacity on the host in bytes. It
+    // should be noted that the host can lie.
+    "remainingstorage": 35000000000, // bytes
+
+    // The smallest amount of data in bytes that can be uploaded or
+    // downloaded when performing calls to the host.
+    "sectorsize": 4194304, // bytes
+
+    // The total amount of storage capacity on the host. It should be noted
+    // that the host can lie.
+    "totalstorage": 35000000000, // bytes
+
+    // The unlock hash is the address at which the host can be paid when
+    // forming file contracts.
+    "unlockhash": "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789ab",
+
+    // The storage proof window is the number of blocks that the host has
+    // to get a storage proof onto the blockchain. The window size is the
+    // minimum size of window that the host will accept in a file contract.
+    "windowsize": 144, // blocks
+
+    // The maximum amount of money that the host will put up as collateral
+    // for storage that is contracted by the renter.
+    "collateral": "57870370370", // hastings / byte / block
+
+    // The maximum amount of collateral that the host will put into a
+    // single file contract.
+    "maxcollateral": "100000000000000000000000000000",  // hastings
+
+    // The price that a renter has to pay to create a contract with the
+    // host. The payment is intended to cover transaction fees
+    // for the file contract revision and the storage proof that the host
+    // will be submitting to the blockchain.
+    "contractprice": "30000000000000000000000000", // hastings
+
+    // The price that a renter has to pay when downloading data from the
+    // host.
+    "downloadbandwidthprice": "250000000000000", // hastings / byte
+
+    // The price that a renter has to pay to store files with the host.
+    "storageprice": "231481481481", // hastings / byte / block
+
+    // The price that a renter has to pay when uploading data to the host.
+    "uploadbandwidthprice": "100000000000000", // hastings / byte
+
+    // The revision number indicates to the renter what iteration of
+    // settings the host is currently at. Settings are generally signed.
+    // If the renter has multiple conflicting copies of settings from the
+    // host, the renter can expect the one with the higher revision number
+    // to be more recent.
+    "revisionnumber": 0,
+
+    // The version of external settings being used. This field helps
+    // coordinate updates while preserving compatibility with older nodes.
+    "version": "1.0.0"
+  },
+
+  // The financial status of the host.
+  "financialmetrics": {
+    // Number of open file contracts.
+    "contractcount": 2,
```

Comment:
For reference the corresponding commit that added this field is 4fa09ed31580ca543ce8ee7df64595e94e426ace

I'm not familiar with the host, but `ContractCount` is decremented in the `removeStorageObligation` method, which to me indicated that it might be only the number of current contracts.